src/git.js

```diff
@@ -156,6 +156,17 @@ module.exports = function gitFactory(repository = '', origin = 'origin') {
                 .then(tags => tags && tags.all && tags.all.indexOf(tagName) > -1);
         },
 
+        /**
+         * Get the latest tag
+         *
+         * @param {String} tagName
+         * @returns {Promise<Boolean>}
+         */
+        getLastTag() {
+            return git(repository).tags()
+                .then( tags => tags && tags.latest);
```

Comment:
I don't remember if the tags are fetched before, to be sure the last tag is the last tag of _origin_ and not the local repo

Reply:
There is a pull from target release branch, so all the tags should be there